doc/source/release-notes/Percona-Server-5.6.36-82.0.rst

```diff
@@ -0,0 +1,86 @@
+.. rn:: 5.6.36-82.0
+
+============================
+|Percona Server| 5.6.36-82.0
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.36-82.0 on
+May 12, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.36-82.0/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.36
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-36.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.34-79.1 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.36-82.0
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.36-82.0>`_.
+
+New Features
+============
+
+ |Percona Server| 5.6 packages are now available for Ubuntu 17.04 (*Zesty
+ Zapus*).
+
+ |Percona Server| now supports :ref:`prefix_index_queries_optimization`.
+
+ :variable:`tokudb_dir_cmd` can now be used to :ref:`edit the TokuDB
+ <editing_tokudb_files_with_tokudb_dir_cmd>` files.
+
+Bugs Fixed
+==========
+
+ Deadlock could occur in I/O-bound workloads when server was using several
+ small buffer pool instances in combination with
```

Comment:
several buffer pool instances in combination with small redo log files and variable ...